scipy/signal/_max_len_seq.py

```diff
@@ -72,6 +72,7 @@ def max_len_seq(nbits, state=None, length=None, taps=None):
 
     MLS has a white spectrum (except for DC):
 
+    >>> import numpy as np
     >>> import matplotlib.pyplot as plt
     >>> from numpy.fft import fft, ifft, fftshift, fftfreq
```

Comment:
Minor nit: maybe we want to keep `np.fft` here instead of importing again? Not sure if that would be too verbose for the code, though.